driver-sync/src/test/functional/com/mongodb/client/AbstractUnifiedTest.java

```diff
@@ -414,11 +415,18 @@ public Void execute() {
                         }
                     } else if (operationName.equals("assertDifferentLsidOnLastTwoCommands")) {
                         List<CommandEvent> events = lastTwoCommandEvents();
-                        assertNotEquals(((CommandStartedEvent) events.get(0)).getCommand().getDocument("lsid"),
+                        String eventsJson = commandListener.getCommandStartedEvents().stream()
```

Comment:
Did you want to keep these changes, or was it just for debugging?

Reply:
I opted to keep them as they are just for the error message should it fail.